source/linear-algebra/source/01-LE/02.ptx

```diff
@@ -192,44 +213,58 @@ Multiply a row by a nonzero constant, for example:
 <sidebyside>
     <p>
         <me>
-            \left[\begin{array}{cc|c}  1 &amp; 2 &amp; 3 
-                                    \\ 4 &amp; 5 &amp; 6 \end{array}\right]
+            \left[\begin{array}{cc|c}  1 &amp; 2 &amp; 4 
+                                    \\ 1 &amp; 3 &amp; 5 \end{array}\right]
         \sim
-            \left[\begin{array}{cc|c} 3 &amp; 6 &amp; 9 
+            \left[\begin{array}{cc|c} 3(1) &amp; 3(2) &amp; 3(4) 
                                     \\ 4 &amp; 5 &amp; 6 \end{array}\right]
         </me>
     </p>
     <p>
         <md>
             <mrow>x  +2y \amp= 3 \amp 3x + 6y \amp= 9</mrow>
-            <mrow>4x + 5y \amp= 6 \amp 4x  +5y \amp= 6 </mrow>
+            <mrow>x + 3y \amp= 5 \amp x  +3y \amp= 5 </mrow>
         </md>
     </p>
 </sidebyside>
+<p>
+    <ol marker="A." cols="2">
+        <li>Solutions must be the <em>same</em>.</li>
+        <li>Solutions might be <em>different</em>.</li>
+    </ol>
+</p>
         </statement>
+        <answer>A. Same</answer>
     </task>
     <task>
         <statement>
             <p>
-Add a constant multiple of one row to another row, for example:
+Subtract one row by another row, for example:
```

Comment:
I wanted to make this a little easier than the general case, and make an interactive for the general case.

Reply:
I think keeping it adding might be cleaner.

Reply:
Done in 4810968